test/servicemesh.go

```diff
@@ -139,7 +135,7 @@ func AllowFromServingSystemNamespaceNetworkPolicy(namespace string) *networkingv
 						{
 							NamespaceSelector: &metav1.LabelSelector{
 								MatchLabels: map[string]string{
-									KnativeSystemNamespaceKey: "true",
+									"knative.openshift.io/part-of": "openshift-serverless",
```

Comment:
maybe reuse the constants? eg. `common.ServerlessCommonLabelKey: common.ServerlessCommonLabelValue`